erpnext/docs/user/manual/en/human-resources/daily-work-summary.md

```diff
@@ -1,15 +1,24 @@
 # Daily Work Summary
 
-Daily Work Summary is way to get a automated way to get a summary of work done by all employees in a company. Once you set it up, each active Employee of the Company gets an email asking them what did they work on during the day.
+Daily Work Summary is way to get a automated way to get a summary of work done by users.
+
+Replies of all users who choose to respond is collected and sent as a summary at midnight. Emails are only sent based on the Holiday List selected for the setting 
+
+**Note:**
+> You must have one active incoming email account setup for this to work.
 
-Replies of all employees who choose to respond is collected and sent as a summary at midnight. Emails are only sent based on the Holiday List setup in the Company or Employee master.
 
-*Note:* You must have one active incoming email account setup for this to work.
 
 ### How to Use
 
-Go to "Daily Work Summary Settings" via HR module or search bar and set the company for which you want to activate this feature.
+Go to "Daily Work Summary Setting" via HR module or search bar and set the users for whom you want to send the reminder.
+
+You can set multiple settings with different set of _users_ from your user list with different _time to send emails_ and with seperate _holiday list_ for each.
+
+You can also choose to customize the _Message_ you send to users.
 
-You can also choose to Customize the Message you send to your employees:
+**Note:**
+>1. If no holiday list is seleted then the email will be sent every day.
+>2. Setting name will be sent as the title of daily summary email.
+>3. Deactivated settings will be skipped from daily check.
```

Comment:
What does this mean?

Reply:
https://github.com/surajshetty3416/erpnext/commit/102cd445a34e0c1116ec5576a4fd0592a6297f5f